make-up/code/5163-mihaela-milosevska/src/app/models/client.ts

```diff
@@ -0,0 +1,17 @@
+export interface Recipe {
```

Comment:
+5 points for the models, the definitions are ok. (`Ingredient` would be a better name than `Ingreditents`, but ok)